btree/btree.c

```diff
@@ -2280,6 +2281,8 @@ static int64_t bnode_get(struct node_op *op, struct td *tree,
 			 */
 			ndlist_tlist_del(op->no_node);
 			ndlist_tlist_add(&btree_active_nds, op->no_node);
+			lru_space_used -= (m0_be_chunk_header_size() +
```

Comment:
If possible check if this is a root node in which case you should skip subtracting the chunk_header_size() and just subtract the node size.

Reply:
Root node will be put in the lru list only by destroy or close tick and mostly it's node descriptor will be removed from lru list and freed up immediately by node_put code(new code added for root node). So do we still need check for root node here?

Reply:
In that case it is not needed